extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -194,65 +214,69 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
     return { mimeType, data };
   };
 
-
-  private doSendMsg = async (msg: SendableMsg) => {
+  private attemptSendMsg = async (msg: SendableMsg): Promise<GmailRes.GmailMsgSend> => {
     // if this is a password-encrypted message, then we've already shown progress for uploading to backend
     // and this requests represents second half of uploadable effort. Else this represents all (no previous heavy requests)
+    // todo: this isn't correct when we're sending multiple messages
```

Comment:
@tomholub Looks like this concept is totally broken. Should I file a separate issue for it?

Reply:
Sure - we'll adjust that, in separate issue, after we collect initial customer feedback.